src/js/components/List/List.js

```diff
@@ -39,6 +40,9 @@ const StyledList = styled.ul`
 
 const StyledItem = styled(Box)`
   ${props => props.onClick && `cursor: pointer;`}
+  ${props =>
+    props.draggable &&
+    `cursor: move;`}
```

Comment:
Looking ahead at designs for ordering of items did we want the cursor to be `cursor: grab`?

Reply:
Eric and I talked about this and decided to use the `move` cursor instead. Typically, `move` is used more for moving items around (look at github's backlog page as an example.) Hand and grab are more for when you're manipulating something that will change it's properties like edit, drop in a trashcan, etc.